src/posix.mak

```diff
@@ -31,6 +31,7 @@ endif
 
 INSTALL_DIR=../../install
 SYSCONFDIR=/etc
+TMP?=/tmp
```

Comment:
For my edification, why `?=` instead of `=`? Does that mean the environment `TMP` overrides it? If you used `=` would you have overridden the enviroment? Thx!